src/data/privacy_de.json

```diff
@@ -15,7 +15,7 @@
                 "Willkommen auf der offiziellen Website der Corona-Warn-App der Bundesregierung.",
                 "<b>1. Verantwortlicher und Datenschutzbeauftragter</b>",
                 "Verantwortlich für die Verarbeitung Ihrer personenbezogenen Daten gemäß den Vorschriften der Datenschutzgrundverordnung (DSGVO) und des Bundesdatenschutzgesetzes (BDSG) bei Besuch dieser Website ist:",
-                "Robert Koch-Institut (RKI)<br />Nordufer 20<br />13353 Berlin<br />vertreten durch den Präsidenten Prof. Dr. L. H. Wieler",
+                "Robert Koch-Institut (RKI)<br />Nordufer 20<br />13353 Berlin<br />vertreten durch den Präsidenten i.V. Prof. Dr. Schaade",
```

Comment:
written out this means:

"vertreten durch den Präsidenten in Vertretung Prof. Dr. Schaade"

which sounds a bit strange!

Does this mean that Prof. Dr. Schaade is not the real President and he is only standing in for him?

Reply:
@larswmh 

Yes, thanks for asking. I have hidden my comment now as "resolved".